### PR TITLE
Add TaskAccessIntegrationTests and fix found problems

### PR DESCRIPTION
**Pull request by katarzynalatos**

### CODE REVIEW

File: ScrumIt/DataAccess/SprintAccess.cs [hunk 1]
```diff
@@ -155,6 +152,19 @@ public static bool CreateNewSprintForProject(SprintModel addedSprint, int projec
...
+                };
+                using (var reader = cmd.ExecuteReader())
+                {
+                    while (reader.Read())
+                    {
+                        addedSprint.SprintId = (int)reader[0];
+                        break;
+                    }
+                }
                 if (result != 1) return false;
```
Comment (pawelaugustyn) [inline ScrumIt/DataAccess/SprintAccess.cs]: That "if" clause should be placed right after line 154 (var result = ...)

File: ScrumIt/DataAccess/TaskAccess.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@ public static List<TaskModel> GetProjectTasksByProjectId(int projectid)
             var tasks = new List<TaskModel>();
             using (new Connection())
             {
-                var cmd = new NpgsqlCommand("select tsk.* from tasks tsk join sprints spr using(sprint_id) where spr.project_id = @projectid order by spr.sprint_id;")
+                var cmd = new NpgsqlCommand("select tsk.* from tasks tsk join projects pr using(project_id) where pr.project_id = @projectid order by pr.project_id;")
```
Comment (pawelaugustyn) [inline ScrumIt/DataAccess/TaskAccess.cs]: select tsk.* from tasks tsk join projects pr using(project_id) where pr.project_id = @projectid order by pr.project_id UNION ALL select tsk.* from tasks tsk join sprints spr using(sprint_id) where spr.project_id = @projectid order by spr.sprint_id;

Look at the method TaskAccess.AssignFromBacklogToSprint - after assigning to sprint we set project_id to 0. We can switch SQL to the one provided above or change the method to not de-assign from project (don't change project_id in task)

File: ScrumIt/DataAccess/SprintAccess.cs (hunk 1 above)
Comment (katarzynalatos) [reply ScrumIt/DataAccess/SprintAccess.cs]: You are right! I have fixed it already. @pawelaugustyn  if everything else is ok, can you approve my pull request?

File: ScrumIt/DataAccess/SprintAccess.cs [hunk 3]
```diff
@@ -155,7 +152,22 @@ public static bool CreateNewSprintForProject(SprintModel addedSprint, int projec
                 cmd.Parameters.AddWithValue("start", startDate);
                 cmd.Parameters.AddWithValue("end", endDate);
                 var result = cmd.ExecuteNonQuery();
-                if (result != 1) return false;
+
```
Comment (salvatien) [inline ScrumIt/DataAccess/SprintAccess.cs]: What is the purpose of this part?

File: ScrumIt/Forms/AddTask.cs [hunk 4]
```diff
@@ -103,17 +103,15 @@ private void addTaskButton_Click(object sender, EventArgs e)
 
         private void priorityTextBox_KeyPress(object sender, KeyPressEventArgs e)
         {
```
Comment (salvatien) [inline ScrumIt/Forms/AddTask.cs]: Is the dot for sure a part of the other 2 conditions?

**Pull request merged**